manifest.toml

```
@@ -2,135 +2,158 @@
+
+        [resources.sources.ui-library]
+        url = "https://github.com/pkp/ui-library/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "4208baa4df7bcc0c38e4a14d17cd27f2ea1bc852cd07348d181f52c44d93b743"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.citationStyleLanguage]
+        url = "https://github.com/pkp/citationStyleLanguage/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "03f4e7f15b56f5e83554a78eb73d5fc6cca3b5b0eb3ebdf33125edf36e78d535"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.COUNTER]
+        url = "https://github.com/pkp/COUNTER/archive/refs/heads/main.zip"
+        sha256 = "a69d7d8e7c1eb12e92db32402cb67b321776000899d925bb22c0d7dbcb6ac3b8"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.browse]
+        url = "https://github.com/pkp/browse/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "a6fcc04acb55b35ef631c319a18b2e00c3e736a630d03218f3415c05fef44ddb"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.googleScholar]
+        url = "https://github.com/pkp/googleScholar/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "117b502c37b44b709cce17f64eb048cd3d8af47f6fa21b0216734c9171e5dad3"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.crossref]
+        url = "https://github.com/pkp/crossref-ojs/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "a7fd22be4dc2e484fc3922ab46acccf9b9c554e058ba1db4731e7f4f4fc35fdb"
+        autoupdate.strategy = "latest_github_tag"
+
+        [resources.sources.webFeed]
+        url = "https://github.com/pkp/webFeed/archive/refs/tags/3_4_0-7.tar.gz"
+        sha256 = "c4a79c10cd79b8329dd20a3d10c595c4f38efdaad3f3e3e195db89145826e70a"
+        autoupdate.strategy = "latest_github_tag"
 
     [resources.system_user]
-    # This will provision/deprovision a unix system user
+    allow_email = true
 
     [resources.install_dir]
-    # This will create/remove the install dir as /var/www/$app
-    # and store the corresponding setting $install_dir
 
     [resources.data_dir]
-    # This will create/remove the data dir as /home/yunohost.app/$app
-    # and store the corresponding setting $data_dir
 
     [resources.permissions]
-    # This will configure SSOwat permission for $domain/$path/
-    # The initial allowed group of user is configured via the init_main_permission question (public=visitors, private=all_users)
     main.url = "/"
 
-    [resources.ports]
-    # This will pick a random port for reverse-proxying and store it as the $port setting
-
     [resources.apt]
-    # This will automatically install/uninstall the following apt packages
-    # and implicitly define the $phpversion setting as 8.0 (if phpX.Y-foobar dependencies are listed)
-    packages = "mariadb-server, deb1, deb2, php8.0-foo, php8.0-bar"
+    packages = "mariadb-server, php8.3-mysqli, php8.3-mbstring, php8.3-xml, php8.3-intl"
```

```suggestion
    packages = "mariadb-server, php8.3, php8.3-mysqli, php8.3-mbstring, php8.3-xml, php8.3-intl"
```

[The CI](https://ci-apps-bookworm-dev.yunohost.org/ci/job/3275) complains about `phpversion` not being set. Let's try to properly declare it from the APT packages.